packages/components/src/radio/tests/chromatic/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,238 @@
+                        <Radio value="manager">Manager</Radio>
+                    </RadioList>
+                </RadioGroup>
+            </Inline>
+            <h1>Bordered with Description</h1>
+            <Inline alignY="start">
+                <RadioGroup {...props} size="sm" variant="bordered">
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <RadioField>
+                            <Radio value="developer">Developer</Radio>
+                            <Text slot="description">Team Lead</Text>
+                        </RadioField>
+                        <Radio value="designer">Designer</Radio>
+                        <RadioField>
+                            <Radio value="manager">Manager</Radio>
+                            <Text slot="description">Team Manager</Text>
+                        </RadioField>
+                    </RadioList>
+                    <HelperMessage>Select one to continue</HelperMessage>
+                </RadioGroup>
+                <RadioGroup {...props} size="md" variant="bordered">
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <RadioField>
+                            <Radio value="developer">Developer</Radio>
+                            <Text slot="description">Team Lead</Text>
+                        </RadioField>
+                        <Radio value="designer">Designer</Radio>
+                        <RadioField>
+                            <Radio value="manager">Manager</Radio>
+                            <Text slot="description">Team Manager</Text>
+                        </RadioField>
+                    </RadioList>
+                    <HelperMessage>Select one to continue</HelperMessage>
+                </RadioGroup>
+            </Inline>
+            <h1>Bordered & Invalid</h1>
+            <Inline alignY="start">
+                <RadioGroup {...props} size="sm" variant="bordered" isInvalid>
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <Radio value="developer">Developer</Radio>
+                        <Radio value="designer">Designer</Radio>
+                        <Radio value="manager">Manager</Radio>
+                    </RadioList>
+                    <ErrorMessage>This field is required</ErrorMessage>
+                </RadioGroup>
+                <RadioGroup {...props} size="md" variant="bordered" isInvalid>
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <Radio value="developer">Developer</Radio>
+                        <Radio value="designer">Designer</Radio>
+                        <Radio value="manager">Manager</Radio>
+                    </RadioList>
+                    <ErrorMessage>This field is required</ErrorMessage>
+                </RadioGroup>
+            </Inline>
+            <h1>Styling</h1>
```

Comment:
We should create a different styling test. This one triggers accessiblity issues. Usually, when story are splitted, i disabled the accessiblity rule for this story. But since it's part of the whole group, we need to change this test a bit

Reply:
There is also an issue with Zoom. Maybe we separate them in a different story and disable the accessibiltiy tests

Reply:
The same thing happens in the checkbox group story. I didn't notice it at first because of the other accessibility errors